transmit-lib/src/cmr/transmit/metadata_db.clj

```diff
@@ -331,7 +331,15 @@
                                 :throw-exceptions false
                                 :headers (ch/context->http-headers context)}))
         status (int (:status response))
-        body (json/decode (:body response))
+        ;; For CMR-4841 - log the first 255 characters of the response body if
+        ;; the parsing of the html throws exception. 
+        response-body (:body response)
+        response-body-logged (subs response-body 0 (min 255 (count response-body)))
+        body (try
+               (json/decode response-body)
+               (catch Exception e 
```

Comment:
The exception is too general. It should be limited to just the JsonParseException that is in the stack trace.

Reply:
Yes. Thanks!